v0/destinations/mp/data/MPProfilePropertiesAndroid.json

```diff
@@ -28,7 +28,10 @@
     "destKey": "$android_lib_version"
   },
   {
-    "sourceKeys": "properties.androidBrand",
+    "sourceKeys": [
+      "context.device.manufacturer",
```

Comment:
Lets keep original order for backward compatibility

Reply:
done